core/src/test/java/io/grpc/internal/BinaryLogProviderTest.java

```diff
@@ -92,4 +289,21 @@ protected int priority() {
       return priority;
     }
   }
+
+  private static final class IntegerMarshaller implements MethodDescriptor.Marshaller<Integer> {
```

Comment:
Rename this to `CountingMarshaller` or `InvocationCountMarshaller` or similar to avoid confusion?